9-ComandiAvanzati/res/slides/15-Soluzioni1.tex

```diff
@@ -0,0 +1,23 @@
+\begin{frame}[fragile]{Soluzioni degli esercizi}
+
+\begin{block}{Esercizio 1a - Concatenazione di stringhe}
+
+\begin{lstlisting}
+\newcommand{\myConcat1}[2] {
```

Comment:
Non funziona, non si possono mettere cosi i numeri nei nomi delle funzioni definite in teoria